packages/ui-components/src/icon/labicon.tsx

```diff
@@ -459,6 +459,7 @@ export class LabIcon implements LabIcon.ILabIcon, VirtualElement.IRenderer {
           title,
           slot,
           tag = 'div',
+          slot,
```

Comment:
You should probably remove this attribute as it is already mentioned at line 460

```suggestion
```